code/script.js

```diff
@@ -1,9 +1,31 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
+
+
+// function toggleFAQ will toggle between class active and without class active
+// active class is styled in CSS (does not exist in HTML)
+// in CSS .active {} will style the question
+// .active + .answer {} display and style the answer 
+
+function toggleFAQ() {
   this.classList.toggle("active")
 }
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+
+// variables
+
+var section1 = document.getElementById("section1")
```

Comment:
If possible try to use 1) `const`, 2) `let` 3) `var` when declaring your variables. For this variable it would be great as a `const`! If you want to know more in detail why, then here's a [article](https://tylermcginnis.com/var-let-const/)!  
```suggestion
const section1 = document.getElementById("section1")
```